### PR TITLE
Add toHTML method

### PR DESCRIPTION
**Pull request by qfox**
Replaces #2 

cc @Yeti-or @voischev 
cc @blond

### CODE REVIEW

**Comment by voischev** [comment]
LGTM

File: index.js [hunk 1]
```diff
@@ -54,9 +56,62 @@ module.exports = function (options, engine) {
...
+        }
+        return callback(null, file);
+      }))
+      .pipe(geval())
+      .pipe(through.obj(function(file) {
+        if (file.isNull()) {
+          return callback(null, file);
+        }
+
+        var html = tryCatch(_ => file.data.apply(bemjsonFile.data), callback);
```
Comment (Yeti-or) [inline index.js:95]: why `_` is better then () ?

Comment (qfox) [inline index.js:95]: I swear, I'll fix this in the next commit with eslint ;-)

File: index.js [hunk 2]
```diff
@@ -54,9 +56,62 @@ module.exports = function (options, engine) {
...
+      return callback(new PluginError(pluginName, 'Parameter should be a Stream'));
+    }
+
+    // Handle multiple templates case
+    var n = 0;
+
+    tmpl
+      .pipe(through.obj(function(file, encoding, callback){
+        if (file.isStream()) {
+          return callback(new PluginError(pluginName, 'Substreaming not supported'));
```
Comment (Yeti-or) [inline index.js:85]: Are you sure that this check must be here? maybe in gulp-eval?

Comment (qfox) [inline index.js:85]: Maybe in both places.

File: index.js [hunk 3]
```diff
@@ -54,9 +56,62 @@ module.exports = function (options, engine) {
...
+          path: name + (n-- || '') + '.html',
+          contents: new Buffer(html)
+        });
+
+        return callback(null, newFile);
+      }));
+    }));
+};
+
+function tryCatch(fn, cb) {
```
Comment (Yeti-or) [inline index.js:111]: we need to make left-pad of this helper =)

**Comment by Yeti-or** [comment]
🆗

**Pull request merged**